keyboards/dztech/dz65rgb/keymaps/adi_dz65rgb_ansi/readme.md

```diff
@@ -0,0 +1,13 @@
+# Adi's DZ65RGB Layout
+[Layout Visual](https://i.imgur.com/xuFkvVI.png)
+
+This layout modifies the extra row on the right and makes some of the key bindings easier to reach on Layer 1.
+## Step by Step
+1. Build your Firmware: run build command in directory
+```make dztech/dzrgb:adi_dz65rgb_ansi```
```

Comment:
- The keyboard name is wrong.
- The `make` command for this board requires you to specify the revision of the board for which you're compiling.

```suggestion
```make dztech/dz65rgb/v1:adi_dz65rgb_ansi```
```

Yours is `/v1` because it takes `.bin` files for firmware.